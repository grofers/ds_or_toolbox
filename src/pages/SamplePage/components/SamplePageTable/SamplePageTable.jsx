import React from "react";

const SamplePageTable = () => <section>Sample Table</section>;

export default SamplePageTable;
