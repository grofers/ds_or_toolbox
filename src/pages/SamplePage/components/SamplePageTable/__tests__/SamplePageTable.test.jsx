import React from "react";
import { render } from "@testing-library/react";
import SamplePageTable from "../SamplePageTable";

describe("SamplePageTable", () => {
  it("should match the snapshot on page load", () => {
    const { asFragment } = render(<SamplePageTable />);

    expect(asFragment()).toMatchSnapshot();
  });
});
