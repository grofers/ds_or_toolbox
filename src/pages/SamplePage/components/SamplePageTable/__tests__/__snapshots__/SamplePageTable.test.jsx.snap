// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SamplePageTable should match the snapshot on page load 1`] = `
<DocumentFragment>
  <section>
    Sample Table
  </section>
</DocumentFragment>
`;
